tardis_env3.yml

```diff
@@ -63,6 +63,7 @@ dependencies:
   - docopt
 
   # Code quality
+  - pre-commit
```

Comment:
We do not currently have a robust method for determining new dependencies to be added. You can expect this to take some time.

Reply:
Determining? Do you mean approving?
If it's the former I'm not sure what to determine here, `pre-commit` is the package required to install/run pre-commit hooks, it's available on [conda-forge](https://anaconda.org/conda-forge/pre-commit), it installs and works fine.
If it's the latter I thought #1563 was about it. Then I suppose I'd need to move this to separate PR, would love to hear if it's the case.

Reply:
Yeah sorry, approving is what I meant. Like I said, not a problem with this PR it will just need to be discussed with tardis core.